tests/unit_tests/test_wave_06.py

```diff
@@ -108,8 +124,22 @@ def test_swap_best_by_category_reordered():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result == True 
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    # refactored
+    assert all(item in [item_d, item_e, item_c] for item in jesse.inventory)
+    assert all(item in [item_a, item_b, item_f] for item in tai.inventory)
```

Comment:
You could simplify this with one time like this as well: `assert [item_d, item_e, item_c] in jesse.inventory`